src/main/java/com/artipie/Settings.java

```diff
@@ -65,6 +67,13 @@
      */
     YamlMapping meta();
 
+    /**
+     * Repo configs storage key, or, in file system storage terms, subdirectory where repo
+     * configs are located relatively to the storage.
+     * @return Repo configs key
+     */
+    Optional<Key> repoConfigs();
```

Comment:
@olenagerasimova it seems that in every case `Key` returned by this method is composed with `storage()` method result to produce a `SubStorage`. That makes me think it would be better to return `Storage` here and put `SubStorage` creation inside